### DISCUSSION TITLE
Calendar view

### DISCUSSION THREAD

**Discussion by GabeDuarteM** [garethgeorge/backrest#426] [Ideas]
I don't think I've seen this on other backup solutions yet, but I think it would be nice to have some sort of calendar view where we could see all currently scheduled tasks, maybe even with their average duration. Aside from the nice visual overview, we could use this to plan when to schedule new tasks, if theres any overlap, things like these

**Comment by semiligneous** [comment]
@GabeDuarteM - I came here looking for this exact feature. I have 20+ plans now (1 for each docker container/app) and I schedule them for different times about 10 minutes apart to avoid conflicts as well as to give time for docker down/up as hooks before/after, but I am losing track of when I have set them, and at some point will run out of time if I give them all 10 minutes each!

@garethgeorge  Being able to see the run schedule as a calendar, with ability to "zoom" in and out (month view, week view, day view where you see the actual hours of schedule more clearly) would be an awesome addition in my opinion!

Some plans no doubt only take a few seconds, so being able to see max, min and average run time would be amazing as a secondary benefit of calendar view. In theory though, that could be implemented independently to calendar view, and still have benefit viewing it elsewhere in the UI.
